src/loaders/loaderUtils.test.ts

```diff
@@ -35,22 +37,28 @@ describe("loaderUtils correlateTopicsWithSchemaSubjects() test", () => {
   });
 });
 
-describe("loaderUtils fetchSubjects() tests", () => {
-  let sandbox: sinon.SinonSandbox;
+describe("loaderUtils fetchSubjects() and fetchSchemaSubjectGroup() tests", () => {
+  // Common suite and setup for loaderUtils functions that interact with SubjectsV1Api.
 
+  let sandbox: sinon.SinonSandbox;
   let listSubjectsStub: sinon.SinonStub;
+  let listVersionsStub: sinon.SinonStub;
+  let getSchemaByVersionStub: sinon.SinonStub;
 
   beforeEach(() => {
     sandbox = sinon.createSandbox();
 
     listSubjectsStub = sandbox.stub();
+    listVersionsStub = sandbox.stub();
+    getSchemaByVersionStub = sandbox.stub();
 
     const mockSubjectsV1Api = {
       list: listSubjectsStub,
+      listVersions: listVersionsStub,
+      getSchemaByVersion: getSchemaByVersionStub,
     };
```

Comment:
Could also go with a `StubbedInstance<SubjectsV1Api>` - same is done in here so we don't have to stub every request method:
https://github.com/confluentinc/vscode/blob/f62afdc3ac02be8d6b152953579d6b9b47375a1a/src/authz/schemaRegistry.test.ts#L16-L33